authorizations/authorization-ranger/build.gradle.kts

```diff
@@ -50,11 +56,9 @@ dependencies {
     exclude("org.apache.ranger", "ranger-plugin-classloader")
     exclude("net.java.dev.jna")
     exclude("javax.ws.rs")
+    exclude("org.eclipse.jetty")
   }
-  implementation(libs.javax.ws.rs.api)
-  implementation(libs.javax.jaxb.api) {
-    exclude("*")
-  }
+  implementation(libs.rome)
```

Comment:
Why do we need to introduce some dependencies like this?

Reply:
Because I excluded some Hive catalog dependencies, RangerHiveIT encountered a ClassNotFoundException, so I introduced these dependencies to fix it.

I think the root cause is that the AuthRanger plugin uses the same class loader as the Hive catalog.  Previously, the Ranger plugin used dependencies of the Hive catalog.